native/native_test.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"gopkg.in/bblfsh/sdk.v1/protocol"
+	"gopkg.in/bblfsh/sdk.v1/sdk/driver"
+)
+
+func TestNative(t *testing.T) {
+	const dir = "../fixtures"
```

Comment:
This is a little redundant considering that the integration tests will also check those files, maybe checking a single string with code and that the native response is Ok is enough here.